src/unity/python/turicreate/toolkits/graph_analytics/pagerank.py

```diff
@@ -35,7 +35,7 @@ class PagerankModel(_ModelBase):
     +-------------------+-----------------------------------------------------------+
     | Field             | Description                                               |
     +===================+===========================================================+
-    | reset_probability | The probablity of random jumps to any node in the graph   |
+    | reset_probability | The probability of random jumps to any node in the graph   |
```

Comment:
Fixing it so the right side of the box lines up would be nice. You should just need to remove a space after "graph".